en/deploy/signup_pythonanywhere.md

```diff
@@ -1,11 +1,15 @@
+PythonAnywhere is a service for running Python code on servers "in the cloud". We'll use
+it for hosting our site, live and on the Internet.
 
-Sign up for a free "Beginner" account on PythonAnywhere:
+Sign up for a "Beginner" account on PythonAnywhere (the free tier is fine, you don't need a credit card).
 
 * [www.pythonanywhere.com](https://www.pythonanywhere.com/)
 
+<img src="images/pythonanywhere_beginner_account_button.png" alt="The PythonAnywher signup page showing button to create a the free 'Beginner' account" />
```

Comment:
Typo: you missed an e in PythonAnywhere.

Reply:
I also thought we weren't supposed to use IMG tags, but instead the ! notation?